libs/sdk-backend-spi/src/common/settings.ts

```diff
@@ -141,6 +141,12 @@ export interface ISettings {
      */
     enableKPIDashboardDrillToURL?: boolean;
 
+    /**
+     * TODO: for development purposes, once feature is done remove.
+     * Indicates whether the drill to URL has filter context transfer enabled.
+     */
+    enableFilterValuesResolution?: boolean;
```

Comment:
Please remove it if it is only for development purposes and not to stay, you can use whatever string with teh settings objects. However if we make a new release of the SDK before this is removed, it would become part of the public API and we could not remove it anymore.